docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc

```diff
@@ -0,0 +1,68 @@
+[id='{context}-pro-working-with-templates-on-rhmap']
+= Working with {Feature} Templates in {ProductShortName}
+
+This section includes:
+
+. xref:{context}-creating-a-new-wfm-project-in-rhmap[Creating a new {Feature} Project in {ProductShortName}]
```

Comment:
See project "DeleteMeWhenever_IAmNotImportant4" on https://testing.zeta.feedhenry.com/#projects/ as to why I have used these names

Reply:
@jstaffor - this names were changed.  Templates on zeta are old.
Here are the actual names (but we can also change them if other suggested)
https://github.com/feedhenry/fh-template-apps/blob/master/global.json#L473-L514

Reply:
I will update these to:

 - WFM Mobile App
 - WFM Portal App
 - WFM Cloud App

and apply in a general review of repoo